workflow/kfdrc_RNAseq_workflow.cwl

```diff
@@ -108,10 +110,11 @@ outputs:
   RNASeQC_counts: {type: File, outputSource: supplemental/RNASeQC_counts, doc: "Contains gene tpm, gene read, and exon counts"}
   kallisto_Abundance: {type: File, outputSource: kallisto/abundance_out, doc: "Gene abundance output from STAR genomic bam file"}
 
+
 steps:
 
   bam2fastq:
-    run: ../tools/samtools_fastq.cwl
+    run: ../new_tools/samtools_fastq.cwl
```

Comment:
Leftover from testing? I don't see this dir.

Reply:
yeah that was just how it was in my research project directory